pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -283,8 +299,27 @@ public void markPartitionForVerification(int partitionGroupId) {
    */
   public long getPartitionIngestionDelayMs(int partitionGroupId) {
     // Not protected as this will only be invoked when metric is installed which happens after server ready
-    Long currentMeasure = _partitionToIngestionTimeMsMap.get(partitionGroupId);
-    return getIngestionDelayMs(currentMeasure);
+    IngestionTimestamps currentMeasure = _partitionToIngestionTimestampsMap.get(partitionGroupId);
+    if (currentMeasure == null) {
+      return 0;
+    }
```

Comment:
Could you add some comments here on when we get into the situation of currentMeasure being null?